h/views/client.py

```diff
@@ -50,23 +116,27 @@ def annotator_token(request):
     return generate_jwt(request, 3600)
 
 
-@view_config(route_name='embed')
-def embed(context, request):
-    request.response.content_type = b'text/javascript'
-    request.response.text = client.render_embed_js(
-        assets_env=request.registry['assets_client_env'],
-        app_html_url=request.route_url('widget'),
-        base_url=request.route_url('index'))
-    return request.response
+@view_config(route_name='embed',
+             renderer='h:templates/embed.js.jinja2')
+def embed(request):
+    request.response.content_type = 'text/javascript'
+
+    assets_env = request.registry['assets_client_env']
+    base_url = request.route_url('index')
```

Comment:
Both names come from the preexisting code, but this is called `service_url` above, I wonder if it'd be worth renaming it to `service_url` here as well? Or even rename both of them to `index_url` to match the route name?